generators/client/templates/react/src/test/javascript/jest.conf.js.ejs

```diff
@@ -3,10 +3,11 @@ module.exports = {
     '^.+\\.tsx?$': 'ts-jest'
   },
   rootDir: '../../../',
-  testMatch: ['<rootDir>/src/test/javascript/spec/**/+(*.)+(spec.ts)'],
+  testMatch: ['<rootDir>/src/test/javascript/spec/**/+(*.)+(spec.ts?(x))'],
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
   moduleNameMapper: {
-    'app/(.*)': '<rootDir>/src/main/webapp/app/$1'
+    'app/(.*)': '<rootDir>/src/main/webapp/app/$1',
+    '\\.(css|scss)$': 'identity-obj-proxy'
```

Comment:
why is this needed?

Reply:
If we don't proxy css modules, then, I noted issue in few test cases like for header.tsx

Reply:
ok